common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -61,7 +61,7 @@ trait ABTestSwitches {
     "Remove the initialisation of Prebid and A9 in the Canada region",
```

Comment:
Looks like @domlander is intending to remove this switch? https://github.com/guardian/frontend/pull/25867

Reply:
Good call! I will revert my change for this PR.